src/components/summary-list/index.md.njk

```diff
@@ -58,9 +61,51 @@ If your summary list does not have any actions, you can choose to remove the sep
 
 To remove borders on a single row, use the `govuk-summary-list__row--no-border` class.
 
+## Summary cards
+
+If you’re showing multiple summary lists on a page, you can show each list within a summary card. This lets you visually separate them and give each list a title and some actions.
+
+Use summary cards when you need to show: 
+- multiple summary lists of the same type, for example a set of applications
```

Comment:
Wonder if using ‘applications’ as an example here might be confusing. Application can have many meanings, depending on context (i.e. job applications versus mobile applications).

After this, we show a list of tenants, but maybe that is too specific. Perhaps we could list a few examples:

```suggestion
- multiple summary lists of the same type, for example a series of interview appointments or application choices
```

(Open to other ideas, point being to include more examples and give some context to them.